src/internal/ajax/ajax.ts

```diff
@@ -268,8 +268,13 @@ export const ajax: AjaxCreationMethod = (() => {
   return create;
 })();
 
+const DOWNLOAD = 'download';
+const LOADSTART = 'loadstart';
+const PROGRESS = 'progress';
+const LOAD = 'load';
+
 export function fromAjax<T>(config: AjaxConfig): Observable<AjaxResponse<T>> {
-  return new Observable<AjaxResponse<T>>((destination) => {
+  return new Observable((destination) => {
```

Comment:
Go TypeScript! Curious is anyone took a pass at trying to remove all inferable generics? cc @cartant

Reply:
Nope. Not yet.